src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -843,6 +846,10 @@
         margin: 0;
         outline-offset: $clr-datagrid-compact-outline-offset;
       }
+
+      .clr-form-control-disabled .datagrid-footer-select.clr-checkbox-wrapper input[type='checkbox']:checked + label {
+        top: 0;
```

Comment:
Just curious, why did the footer "selection count" change? I wouldn't expect it to be impacted.

Reply:
The inherited forms control is overridden above (when not in compact mode). I tried to only use the class but it is so specific above I had to duplicate the same selector here to override this in compact mode. 

Technically its not related to the width calculation but I saw it when working through all of our demos. I can separate it out into a different PR if you want me to.

Reply:
That's okay, you can leave it here. :)